data/language/english_uk.txt

```diff
@@ -4111,6 +4111,19 @@ STR_5802    :{SMALLFONT}{BLACK}Stops guests from littering and vomiting
 STR_5803    :{SMALLFONT}{BLACK}Rotate selected map element
 STR_5804    :Mute sound
 
+#tooltips
+STR_5805    :{SMALLFONT}{BLACK}Use GPU for graphics instead of the CPU
```

Comment:
"… but improve capture compatibility)"

@Gymnasiast can performance be `slower`? I'd rather associate this with velocity and say performance is `lower`, `degraded` or `decreased`. The hit on performance is also not that big, you may want to reflect that in the tip.